requirements.txt

```diff
@@ -0,0 +1,2 @@
+python-dateutil
+pyopenssl
```

Comment:
Missing futures.  May not be needed to build the docs, but we may as well keep in sync with setup.py.